contrib/cirrus/runner.sh

```diff
@@ -398,9 +398,7 @@ dotest() {
 }
 
 _run_machine() {
-    # TODO: This is a manually-triggered task, if that ever changes need to
-    # add something like:
-    # _bail_if_test_can_be_skipped docs test/e2e test/system test/python
+    # N/B: Can't use _bail_if_test_can_be_skipped here b/c content isn't under test/
```

Comment:
You might save a future maintainer some energy by saying "localmachine tests live under pkg/machine/e2e, not under test/" ... but it's fine with me either way.

/lgtm
/hold

Reply:
Thanks @edsantiago